### ISSUE TITLE
os.path.dirname accepts AnyStr and not Traversable

### ISSUE THREAD

**Issue by PiotrAniola82** [IBM/javacore-analyser#74]
This code is flagged as wrong in javacore_analyser_batch.py create_output_files_structure():
```

style_css_resource: Traversable = importlib_resources.files("javacore_analyser") / "data" / "style.css"
data_dir = os.path.dirname(style_css_resource)
```

Unexpected type(s): (Traversable) Possible type(s): (AnyStr) (PathLike[Union[Union[str, bytes], Any]]) 

Same issue in javacore_set.py __create_index_html()

**Comment by kkazmierczyk** [comment]
@PiotrAniola82 this is really confusing and I do not know how to easy fix it.

If I change to 
   ` style_css_resource: PathLike = importlib_resources.files("javacore_analyser") / "data" / "style.css"`

I got the following error:
`Expected type 'PathLike', got 'Traversable' instead`

**Comment by PiotrAniola82** [comment]
Fixed as part of #75
